datalad/distributed/tests/test_create_sibling_gitlab.py

```diff
@@ -77,15 +77,16 @@ def test_dryrun(path):
         site='dummy', sibling='dummy',
     )
     # now a working, fully manual call
-    res = ctlg['root'].create_sibling_gitlab(
-        dryrun=True, on_failure='ignore',
-        site='dummy', project='here',
-    )
-    assert_result_count(res, 1)
-    assert_result_count(
-        res, 1, path=ctlg['root'].path, type='dataset', status='ok',
-        site='dummy', sibling='dummy', project='here',
-    )
+    for p in (None, ctlg['root'].path):
```

Comment:
`p` seems to be not used below -- shouldn't it be?

Reply:
LOL, indeed! Thx for catching that!

Reply:
Test fixed. Pointed to a flaw in the implementation. Adjusted and added a comment pointing out the (avoided) pitfall.

Reply:
Force pushed an amendment.